Makefile

```diff
@@ -5,7 +5,7 @@ DESTINATION ?= "platform=iOS Simulator,name=iPhone 5"
 PROJECT := Segment-Amplitude
 XC_ARGS := -scheme $(PROJECT)_Example -workspace Example/$(PROJECT).xcworkspace -sdk $(SDK) -destination $(DESTINATION) ONLY_ACTIVE_ARCH=NO
 
-install: Example/Podfile $(PROJECT).podspec
```

Comment:
Why is this change needed?

Reply:
It wasn't - just consistent with other Makefiles in other integrations, I can change back - we have to redo Travis or move onto circle - so I will have a project to update Makefile scripts and travis/or circle builds and can address there